tony-core/src/main/java/com/linkedin/tony/tensorflow/TonySession.java

```diff
@@ -161,6 +163,43 @@ public boolean allTasksScheduled() {
     return true;
   }
 
+  public int getTotalTasks() {
+    return jobTasks.values().stream().mapToInt(arr -> arr.length).sum();
```

Comment:
you could use `.reduce` ..

Reply:
I think it's more complicated using `.reduce`:
```
return jobTasks.values().stream().reduce(0, (currTotal, taskArr) -> currTotal + taskArr.length, (count1, count2) -> count1 + count2);
```